planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -1716,6 +1716,34 @@ void GoalPlannerModule::updateSafetyCheckTargetObjectsData(
   goal_planner_data_.ego_predicted_path = ego_predicted_path;
 }
 
+bool GoalPlannerModule::checkSafetyWithRSS(
+  const PathWithLaneId & planned_path,
+  const std::vector<PoseWithVelocityStamped> & ego_predicted_path,
+  const std::vector<ExtendedPredictedObject> & objects, const double hysteresis_factor) const
+{
+  // Check for collisions with each predicted path of the object
+  const bool is_safe = !std::any_of(objects.begin(), objects.end(), [&](const auto & object) {
+    auto current_debug_data = marker_utils::createObjectDebug(object);
+
+    const auto obj_predicted_paths = utils::path_safety_checker::getPredictedPathFromObj(
+      object, objects_filtering_params_->check_all_predicted_path);
+
+    return std::any_of(
+      obj_predicted_paths.begin(), obj_predicted_paths.end(), [&](const auto & obj_path) {
+        const bool is_collision = !utils::path_safety_checker::checkCollision(
```

Comment:
[NITS] I prefer has_collision rathar than is_collision

Reply:
both seem to exist

![image](https://github.com/autowarefoundation/autoware.universe/assets/39142679/facae5f4-2372-477b-933b-6439f0c029c4)
![image](https://github.com/autowarefoundation/autoware.universe/assets/39142679/787b3e00-5fa5-4c8b-8214-f4247fd5bc52)

Reply:
but I also prefer `has` so fixed

840f40850e5f801089114a6f28cf86adddb6ec50